src/index.js

```diff
@@ -271,6 +271,10 @@ exports.init = async (Implementation) => {
       loadCollections(configStore.configDir);
     }
 
+    if (configStore?.Implementation?.RequestUnflattener) {
```

Comment:
[CAUTION] This might change depending on the discussion related to this comment :
ForestAdmin/forest-express-mongoose#644 (comment)